support/modules/rootfs-image

```diff
@@ -30,15 +33,28 @@ else
     passive_num=$MENDER_ROOTFS_PART_A_NUMBER
 fi
 passive_num_hex=$(printf '%x' $passive_num)
+upgrade_available="$(${PRINTENV} upgrade_available)"
+upgrade_available="${upgrade_available#upgrade_available=}"
```

Comment:
I like it, but could just as well be done on the above line, no?

Reply:
I couldn't find a way to combine the `#` operator with a command substitution. I think it requires that it already is in a variable.

Reply:
but does it have to be done via variable substitution? :p

Reply:
I'm sure there are other ways, but what are you suggesting? 🙂

Reply:
uno pipo was what struck me, but it's not necessary. Just leave it be